capstone/client/src/components/StoreDetailCards.js

```diff
@@ -44,7 +44,7 @@ class StoreDetailCards extends Component {
                 <div>{Object.keys(store.items[itemType]).map((index, i) => (
                   <ListItem key = {i}>
                   <ListItemText>
-                    {store.items[itemType][index].itemName} (${store.items[itemType][index].itemPrice})
+                    {store.items[itemType][index].itemName} (${(store.items[itemType][index].itemPrice-.005).toFixed(2)})
```

Comment:
why -0.005?

Reply:
because toFixed is a rounding function, javascript doesnt have a truncate function to my knowledge unless you convert it to a string obviously, just something i found on github